composer.json

```diff
@@ -14,7 +14,8 @@
         }
     ],
     "require": {
-        "php": "^7.1 || ^8.0"
+        "php": "^7.1 || ^8.0",
+        "symfony/deprecation-contracts": "^2.2"
```

Comment:
I think it would be best if we limited the number of dependencies this project has to avoid conflicts for users of this package.

What do you think about copying the function into our repository instead?

Reply:
I am really hesitant about copy features from other packages. Composer is a great tool to handle dependency conflicts. I suggest we can be reactive instead of proactive in this case. Ie, let's see if it become a problem for anyone, if so, we can consider copying it. 

(Im not even sure how this specific package would be able to cause conflicts)

Reply:
@Nyholm 

Right now it would cause conflicts with packages that require `symfony/deprecation-contracts:^1.0.0`.

Reply:
Im not sure why, but there is no 1.x versions. 

https://packagist.org/packages/symfony/deprecation-contracts